### PR TITLE
docs/feat(CHANGELOG): Expand changelog for v3.26.0

### PR DESCRIPTION
**Pull request by joshtrichards**
TLDR: Messing with expanding release changelog to:

- increase coverage of fixes/changes/additions
- make/keep easy to read

TODO:

- [x] First pass that itemizes all non-translation related commits
- [x] Categorize items
- [x] Lightly edit items for readability 
- [ ] Heavily edit for casual reading

---

Editing a changelog to make it presentable, readable, and useful (beyond just skimming commit messages) takes time. This is an experiment to see what's involved, see if it's worthwhile, get some feedback from the community, start somewhere and improve as we go along, etc.

The existing changelog often leaves users with an inappropriate impression of what has changed. Incomplete ones (or ones that aren't easy to comprehend in a relatively quick manner to non-developers) impact trust and confidence.

It's about not only what's been added, but fixed/updated/etc. And bugs and regressions will happen, but they're easier to accept if changes are *not* coming out of a "black box" (I'm talking from the perspective of those that aren't actively involved in the development - or which are only involved in development on the peripheral or elsewhere within @nextcloud).

I further propose that if we decide to keep doing something similar to this, that we:

- add a checkbox to the default PR template (after the "Tests written") that says something like:
  - *[ ] Entry added to CHANGELOG.md for this change
    - in the Unreleased section
    - A preliminary one-liner is fine the changelog editor may tweak it at release time as deemed appropriate
  - This will make life *way* easy for compiling these and disperse the extra work so that it can become more sustainable (that's my hypothesis anyhow) 
- split up release duties to keep things more manageable (loosely; sometimes this will still be the same person):
  - changelog editor
  - builder/deployer/uploader
- have the changelog be 99% done with the first beta/RC
- take the time to continuously improve the "possible to automate better" areas of the release process
- push out these stronger changelogs with betas/RCs to foster/encourage user/community testing (builds excitement and also makes it clearer what areas are the ones likely to need testing)

I'll also include this quote from [Keep a Changelog](https://keepachangelog.com/en/1.1.0/) because it seems relevant: :smile: 

> A changelog which only mentions some of the changes can be as dangerous as not having a changelog. While many of the changes may not be relevant - for instance, removing a single whitespace may not need to be recorded in all instances - any important changes should be mentioned in the changelog. By inconsistently applying changes, your users may mistakenly think that the changelog is the single source of truth. It ought to be. With great power comes great responsibility - having a good changelog means having a consistently updated changelog. 

- [x] Tests written, or not not needed

### CODE REVIEW

**Comment by joshtrichards** [comment]
One area that this work highlighted is that `android-library` could use something similar, but for now the key is to note any key changes that are relevant to the Android client. So presently I can "hack" that effort by simply expanding our changelog here and including the key PRs from the library that are relevant here. e.g. for v3.26 this looks like it should have included:

## Added

- Chunked uploading to S3 via multipart upload (nextcloud/android-library#1173) 
- Add support for metadata: size (width/height) and geolocation (nextcloud/android-library#1178) 

The second was already sort of covered by an existing entry, but the chunked uploading wasn't mentioned in my first pass. It's important though! So I'll have to add that in a follow-up PR to this changelog. :-)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,139 @@
...
+- Commit list for this release: https://github.com/nextcloud/android/compare/stable-3.25.0...stable-3.26.0
+- Some changes were also backported to v3.25 but first made their appearance in this release during development so they're included here
+
+### Added
+
+- Image editor (#11746/#11608)
+- Tighter Nextcloud ecosytem apps integration (#11744/#11811/#11971/#11942)
+- Enhanced image preview / details including map (#11897/#11928)
+- E2EE v1.2 + file drop support + enhanced tests / miscellaneous tweaks (#11630)
+- Extra options for uploading into various patterns of subfolders (#11852/#11642)
```
Comment (joshtrichards) [inline CHANGELOG.md:32]:
```suggestion
- Auto-upload into various date patterns of subfolders (#11852/#11642)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,139 @@
...
+- Some E2EE enhancements
+- Various upstream and dependency updates (which include additional bug fixes and enhancements)
+- Some technical debt cleared (both directly and indirectly)
+- No breaking changes
+- Contributors to this release include: 
+  - @jsoberg, @tobiasKaminsky, @AndyScherzinger, @ZetaTom, @jaller94, @tanganellilore, @skjnldsv, @afflux, @surinder-tsys, @rakekniven, @AlvaroBrey, @nickvergessen, @dean36963, and @AkshayChaurasia98
+  - anyone that tested an early release, reported a bug, proposed an enhancement idea, helped with translations, triaged an issue, reviewed a PR, and/or assisted another community member
+  - the entire Nextcloud community 
+- Release coordinators: @tabiasKaminksy, @AndyScherzinger
+- Changelog editor: @joshtrichards
```
Comment (joshtrichards) [inline CHANGELOG.md:21]:
```suggestion
- Changelog editor for this release: @joshtrichards
```

**Comment by joshtrichards** [comment]
Closing this out. Right now this isn't maintainable as-is. It was an experiment.
